Bot/app/api/meetings.py

```diff
@@ -207,15 +212,16 @@ async def calendar_webhook(
 
     
     for i in range(20):
-        token = redis_client.get(str(x_goog_channel_id))
-        if token: break
+        user_id = redis_client.get(str(x_goog_channel_id))
+        if user_id: break
 
-    if token:
+    if user_id:
         logger.info("Calling /meetings/ endpoint via requests")
         try:
-            response = requests.get(
+            response = requests.post(
                 "http://localhost:8001/meetings/",
```

Comment:
i dont think this should be hardcoded in the code? @Abhi-at-Josh 
Not sure if this will work anywhere apart from local